examples/blog/init.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
-rm -rf ./store
-cp -R ./store-default ./store
+if [[ ! -d ./store ]]; then
```

Comment:
The init script should delete the existing store and create a new one to start from scratch. Maybe an argument to the script could be added. If the argument is given an existing store should not be delete. Maybe `-k` for keep?

Reply:
Hm, my thought was actually that the init should be safe to run multiple times so that you can have it precede the app start. Without the need to run manually